src/mame/bmc/koftball.cpp

```diff
@@ -207,7 +158,31 @@ void koftball_state::video_start()
 	m_tilemap[0]->set_transparent_pen(0);
 	m_tilemap[2]->set_transparent_pen(0);
 
-	m_pixbitmap = std::make_unique<bitmap_ind16>(0x400, 0x200);
+	m_tilemap[1]->set_transparent_pen(0);
+	m_tilemap[3]->set_transparent_pen(0);
+
+	m_pixbitmap = std::make_unique<bitmap_ind16>(512, 256);
```

Comment:
Is there a reason for dynamically allocating the bitmap object itself?  The bitmap classes don’t allocate storage until you call `resize` or `allocate` – this adds an extra layer of indirection.

Reply:
I just reused what was already available in the driver for convenience, and didn't made great efforts in testing this beyond a couple service mode items (i.e. if there's a video register that controls pitch width or if it uses this as priority mask).